tools/pipelines/build-markdown-magic.yml

```diff
@@ -57,17 +57,25 @@ pr:
     - .prettierignore
     - tools/markdown-magic
     - tools/pipelines/build-markdown-magic.yml
-    - tools/pipelines/templates/build-npm-package.yml
+    - tools/pipelines/templates/1ES/build-npm-package.yml
     - tools/pipelines/templates/include-set-package-version.yml
     - tools/pipelines/templates/include-vars.yml
     - tools/pipelines/templates/include-install-pnpm.yml
     - tools/pipelines/templates/include-use-node-version.yml
     - tools/pipelines/templates/include-process-test-results.yml
     - scripts/*
 
+variables:
+  - template: /tools/pipelines/templates/include-vars.yml@self
+    parameters:
+      publishOverride: '${{ parameters.publishOverride }}'
+      releaseBuildOverride: '${{ parameters.releaseBuildOverride }}'
+      buildNumberInPatch: ''
```

Comment:
Nit: can we add a comment explaining why this is empty?

Reply:
I [mentioned this one in another of the PRs](https://github.com/microsoft/FluidFramework/pull/19243#discussion_r1459298019), I suggest we leave it empty for now and we'll address it as part of the post-migration cleanup of the pipelines.